src/wp-includes/block-supports/background.php

```diff
@@ -90,7 +90,7 @@ function wp_render_background_support( $block_content, $block ) {
 			$existing_style = $tags->get_attribute( 'style' );
 			$updated_style  = '';
 
-			if ( ! empty( $existing_style ) ) {
+			if ( is_string( $existing_style ) && ! empty( $existing_style ) ) {
```

Comment:
Note for reviewers: the `empty` on this line would not have any issues, but would do the wrong thing if passed `true`, `! empty( true )` evaluates to `true`, so this block would be entered if if `$existing_style` is `true`, for example with HTML like `<div style>`. Inside the block, `$existing_style` is expected to be a `string` and is passed to string methods.